gradle/libs.versions.toml

```diff
@@ -1,30 +1,29 @@
 [versions]
 androidxCore = "1.13.1"
-androidxLifecycle = "2.8.4"
+androidxLifecycle = "2.8.5"
 kotlinxCoroutinesTest = "1.8.1"
 junit = "4.13.2"
-mockkAndroid = "1.13.9"
+mockkAndroid = "1.13.12"
 jvmTarget = "21"
-kotlin = "2.0.10"
-ksp = "2.0.10-1.0.24"
-gradlePlugin = "8.5.2"
+kotlin = "2.0.20"
+# should match kotlin https://github.com/google/ksp/releases
+ksp = "2.0.20-1.0.25"
+gradlePlugin = "8.6.0"
 googleServices = "4.4.2"
 crashlyticsPlugin = "3.0.2"
-firebaseBOM = "33.1.2"
-manesVersions = "0.44.0"
-littleRobotsCatalogUpdates = "0.8.1"
+firebaseBOM = "33.2.0"
```

Comment:
the `33.3.0` has already been released: https://firebase.google.com/support/release-notes/android#2024-09-12